### ISSUE TITLE
Question: How go deal with version of ROM  ?

### ISSUE THREAD

**Issue by znoxx** [zurdi15/romm#682]
My config file part:
```
system:
  platforms:
    GENESIS: 'genesis-slash-megadrive' # In this example if you have a 'gc' folder, RomM will treat it like the 'ngc' folder
    NES: 'nes' # In this example if you have a 'psx' folder, RomM will treat it like the 'ps' folder
    SNES: 'snes'
  versions:
    mame: 'arcade'
    CPS1: 'arcade'
    CPS2: 'arcade'
    CPS3: 'arcade'
    neogeotest: 'neogeomvs'
```

In MAME folder i have 2 files:
term2.zip
term2la.zip

Those files are treated as Terminator 2 Arcade, which is correct.
But in description I see combobox "Versions", with 2 entries (absolutely correct). 
The problem is -- those values are empty in combobox

![image](https://github.com/zurdi15/romm/assets/6072354/bb82ecb9-e6c3-466a-bb40-3693576488a4)

How can I edit those entries, assigning here values e.g.
rev LA4 08/03/92
rev LA2 12/09/91

Thanks in advance.

**Comment by znoxx** [comment]
Thanks a lot for fast reaction!
Your product is fantastic, especially for handheld emulator owners.
Will use it on regular basis.

**Comment by gantoine** [comment]
Fixed in the latest release

**Comment by znoxx** [comment]
Will give a try in couple of days.
Can you please point to right docker image ?

**Comment by gantoine** [comment]
Sure thing, you can pin romm to `3.0.1`, the fix is in that version

**Comment by znoxx** [comment]
Hi there.
Not sure it worked.

Here is my folder:
![image](https://github.com/zurdi15/romm/assets/6072354/6cbfdc19-c6e8-4ff9-9f1f-a6e8c1967240)

Here is a part of my config:
```
# Associate different platform names to your current file system platform names
system:
  platforms:
    GENESIS: 'genesis-slash-megadrive' # In this example if you have a 'gc' folder, RomM will treat it like the 'ngc' folder
    NES: 'nes' # In this example if you have a 'psx' folder, RomM will treat it like the 'ps' folder
    SNES: 'snes'
    psxtest: 'ps'
    dctest: 'dc'
    neogeotest: 'neogeomvs'
    pcengine: 'turbografx16--1'
  versions:
    mame: 'arcade'
    CPS1: 'arcade'
    CPS2: 'arcade'
    CPS3: 'arcade'
```

docker-compose part:
```
version: '3'
services:
  romm:
    image: zurdi15/romm:3.0.1
    container_name: romm
    environment:
      - ROMM_DB_DRIVER=mariadb # mariadb 
      - ROMM_HOST=https://localhost:3000 # [Optional] your host ip or domain name (including http(s)://, subdomain and port if needed). Being used only for webRcade feed for now.
      - ROMM_AUTH_SECRET_KEY=328b159c2fb10bd285d9cd938153d5f2de5bdef6e8e7836194d59c9bc0d8b014
      - ROMM_AUTH_USERNAME=admin
      - ROMM_AUTH_PASSWORD=admin
      - DISABLE_CSRF_PROTECTION=true
      - DB_HOST=romm_db
      - DB_PORT=3306
      - DB_USER=romm-user
      - DB_NAME=romm # Should match the MYSQL_DATABASE value in the mariadb container
      - DB_PASSWD=rommsql
      - IGDB_CLIENT_ID=xxxxxx  # Fill in the client ID from the previous step
      - IGDB_CLIENT_SECRET=xxxx  # Fill in the client secret from the previous step
      - ENABLE_RESCAN_ON_FILESYSTEM_CHANGE=true 
      - ROMM_HOST=https://localhost:3000 
      - ENABLE_SCHEDULED_UPDATE_MAME_XML=true
      - SCHEDULED_UPDATE_MAME_XML_CRON=0 5 * * *
    volumes:
      - 'library:/romm/library/roms'
      - './resources:/romm/resources'
      - './assets:/romm/assets'
      - './logs:/romm/logs'
      - './config:/romm/config'
      - './romm_redis_data:/redis-data'
```

After scan, both term2.zip and term2la2.zip were attached to same title (Terminator 2). I changed game title to Terminator 2 Arcade (which is correct)

And here is the result:

![image](https://github.com/zurdi15/romm/assets/6072354/1ab384af-c060-4e42-9926-01a4df41ebf3)

Combobox has 2 emtpy values, but they successfully switch the title 

term2.zip:
![image](https://github.com/zurdi15/romm/assets/6072354/433f6469-03e0-486a-80d3-41d89f43f3af)

term2la.zip
![image](https://github.com/zurdi15/romm/assets/6072354/0f5ec750-fec3-4d16-a58d-84ba35d23430)

Honestly, I don't see any changes introduced. Also i cannot see any commit for this issue.

**Comment by gantoine** [comment]
@znoxx I apologize for the confusion, the fix was not released 3.0.1, but will be released in 3.1.

**Comment by kuya1284** [comment]
@gantoine, sorry to resurrect this, but I saw that this issue was fixed and released in v3.1; however, I'm noticing this same issue with v3.7.2. I'm curious if this issue somehow resurfaced and if I should report the issue as a new bug. Here are screenshots showing the issue I'm running into for a CPS game.

![Image](https://github.com/user-attachments/assets/33c6783b-d26c-44e7-b9d6-4019e944c564)

![Image](https://github.com/user-attachments/assets/dc070b3d-d07f-4bb6-b854-200d982eaea9)

If this is a regression and you need me to create a new issue, could you please let me know? Thanks.

**Comment by gantoine** [comment]
In your case it's a different issue, since there are no tag names in the file there's nothing in the UI to distinguish the games. We're making improvements to arcade support in the next month, so wait a bit and this should get better.
